### PR TITLE
Support for offline installation without a Mac machine/macOS

### PR DESCRIPTION
**Pull request by kwadwokyeremeh**
This PR allows users mostly with either bad internet or without Mac PC/macOS to install macOS ventura offline.

### CODE REVIEW

File: run_offline.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### How to run the Installation offline without macOS
+
+- Download the ventura file (InstallAssistant.pkg) from https://mrmacintosh.com/macos-ventura-13-full-installer-database-download-directly-from-apple/(https://mrmacintosh.com/macos-ventura-13-full-installer-database-download-directly-from-apple/)
+- Create an iso file `InstallAssistant.iso` with the downloaded InstallAssistant.pkg and `run_offline.sh` in the `scripts/run_offline.sh` files
```
Comment (kholia) [inline run_offline.md:4]: How are you creating the `InstallAssistant.iso` file?

Comment (kwadwokyeremeh) [reply run_offline.md:4]: using this cmd 
`genisoimage -allow-limited-size -l -J -r -iso-level 3 -V InstallAssistant -o InstallAssistant.iso path/to/InstallAssistant.pkg path/to/OSX-KVM/scripts/run_offline.sh`

**Comment by kholia** [comment]
Thanks!

I have manually pulled in this change now.

**Comment by nesper8** [comment]
is it possible to adapt script to install Big Sur offline?

**Comment by kwadwokyeremeh** [comment]
> is it possible to adapt this script to install Big Sur offline? also it seems that the version of mkisofs which is shipped on archlinux foesnt support the -allow-limited-size flag for some reason

yeah

**Comment by kwadwokyeremeh** [comment]
try this 
genisoimage -allow-limited-size -l -J -r -iso-level 3 -V InstallAssistant -o InstallAssistant.iso path/to/InstallAssistant.pkg path/to/OSX-KVM/scripts/run_offline.sh

> is it possible to adapt this script to install Big Sur offline? also it seems that the version of mkisofs which is shipped on archlinux foesnt support the -allow-limited-size flag for some reason

**Comment by kwadwokyeremeh** [comment]
> is it possible to adapt this script to install Big Sur offline? also it seems that the version of mkisofs which is shipped on archlinux doesnt support the -allow-limited-size flag for some reason

1. yeah
2. 
try this 
`genisoimage -allow-limited-size -l -J -r -iso-level 3 -V InstallAssistant -o InstallAssistant.iso path/to/InstallAssistant.pkg path/to/OSX-KVM/scripts/run_offline.sh`

**Comment by nesper8** [comment]
Like I said
Im unable to use the -allow-limited-size flag on both genisoimage and mkisofs

**Pull request closed without merging**